site/docs/relations/logical_relations.md

```diff
@@ -268,14 +268,23 @@ The set operation encompasses several set-level operations that support combinin
-| Union Distinct          | Returns all the records from each set, removing any rows that are duplicated (within or across sets).         | If a field is nullable in any of the inputs, it is nullable in the output.
-| Union All               | Returns all records from each set, allowing duplicates.                                                       | If a field is nullable in any of the inputs, it is nullable in the output. |
+For some set operations, whether a specific record is included in the output and if it appears more than once depends on the number of times it occurs across all inputs. In the following table, treat:
+* m: the number of time a records occurs in the primary input
+* n1: the number of times a record occurs in the 1st secondary input
+* n2: the number of times a record occurs in the 2nd secondary input
+* ...
+* n: the number of times a record occurs in the nth secondary input
+
+| Operation                  | Description                                                                                                                                                                                                                                                   | Examples                                                                                                                                                         | Output Nullability
+|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------| -----------------------------
+| Minus (Primary)            | Returns all records from the primary input excluding any matching rows from secondary inputs, removing duplicate rows within or across sets.<br/>Each value is treated as a unique member of the set, so duplicates in the first set don’t affect the result. | MINUS<br/>&nbsp;&nbsp;m:  {1, 2, 2, 3, 3, 3, 4}<br/>&nbsp;&nbsp;n1: {1, 2}<br/>&nbsp;&nbsp;n2: {3}<br/>YIELDS<br/>{4}                                            | The same as the primary input.                                                                                                                                                                     
+| Minus (Primary All)        | Returns all records from the primary input excluding any matching records from secondary inputs.<br/>For each specific record returned, the output contains max(0, m - sum(n1, n2, …, n)) copies.                                                             | MINUS ALL<br/>&nbsp;&nbsp;m:  {1, 2, 2, 3, 3, 3, 3}<br/>&nbsp;&nbsp;n1: {1, 2, 3, 4}<br/>&nbsp;&nbsp;n2: {3}<br/>YIELDS<br/>{2, 3, 3}                            | The same as the primary input.
```

Comment:
minor: in your examples
```
MINUS
  m: {1, 2, 2, 3, 3, 3, 4}
  n1: {1, 2}
  n2: {3}
YIELDS
{4}
```

`{1, 2, 2, 3, 3, 3, 4}` is the primary input, but `m` is define as "the number of time a records occurs in the primary input" so I don't think they quite line up based on the definition. I understand what you are trying to do though, which is indicated that
```
  m: {1, 2, 2, 3, 3, 3, 4}  // this is the primary input
  n1: {1, 2}  // this is the 1st secondary input
  n2: {3}     // this is the 2nd secondary input
```